airgen/src/lib.rs

```diff
@@ -341,7 +341,7 @@ impl<'a> ASMPILConverter<'a> {
             .find(|o| o.name == callable)
             .unwrap_or_else(|| {
                 panic!(
-                    "function/operation not found: {}.{}",
+                    "function/operation not found: {}::{}",
```

Comment:
Please double-check, but I think this should stay as `.`, because it accesses the member of an instance (a value) and not a type or module element.